lib/ansible/plugins/action/fetch.py

```diff
@@ -72,7 +72,7 @@ def run(self, tmp=None, task_vars=None):
             source = self._remote_expand_user(source)
 
             remote_checksum = None
-            if not self._play_context.become:
+            if self._connection.become is None:
```

Comment:
Same note here about consistency.  It could easily apply the the change above here as well.